crates/api_common/src/site.rs

```diff
@@ -660,3 +664,25 @@ pub struct BlockInstance {
 pub struct BlockInstanceResponse {
   pub blocked: bool,
 }
+
+#[derive(Debug, Serialize, Deserialize, Clone, Default, PartialEq, Eq, Hash)]
+#[cfg_attr(feature = "full", derive(TS))]
+#[cfg_attr(feature = "full", ts(export))]
+pub struct AdminBlockInstanceParams {
```

Comment:
Yes its a bit awkward with the duplicate names. In case of user instance blocks this is "solved" by naming the db struct `InstanceBlock` and the api struct `BlockInstance` :D

Reply:
I'd be up for renaming all these in a unique way. We name the responses response already, and could name all the forms something else: maybe Param or Action. Can't be Form because the database forms use that.